Several map updates

DevilMudger opened the pull request:
Mostly fixes for spawns that weren't working properly before the clipping bug was fixed

TotallyMehis commented:
Did you add navs that go through the func_brush? And curious, what did you remove from unknownalpha and water canyon?

DevilMudger commented:
Yeah I did. I also fixed little things here and there.

Water canyon I removed the whole of the nav mesh which covered the floor under the river since it's unnecessary. Unknown I removed and redid the nav mesh around the train since zombies no longer clip through it.
